src/cufinufft.cu

```diff
@@ -500,19 +499,19 @@ int cufinufft_default_opts(finufft_type type, int dim, nufft_opts &opts)
 		case 2:
 		{
 			opts.gpu_maxsubprobsize = 1024;
-			if(type == type1){
+			if(type == 1){
```

Comment:
Hey, if you remember could you tag me next time there is an API change like this?  1,2,3 is better, but I didn't realize it changed :) The enum it replaced started at 0. Naturally I thought I had a bug in my side of the code for a while :D..

Reply:
Yes, sure. Sorry for the trouble that it caused... I will make sure to let you know if similar changes made in the future.

Reply:
> Macro tricks is how I, and then Ludvig, made the kernel eval so fast...

Yeah I read that too quickly before I realized what was actually going on... Seems like a pretty neat solution.